### PR TITLE
20346 - Refactor Oracle to point at Datawarehouse instead of CPRD

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/20346

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: jobs/payment-jobs/services/__init__.py [hunk 1]
```diff
@@ -12,4 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """This module holds services used by jobs."""
-from .oracle import oracle_db
+# from .oracle import oracle_db
```
Comment (seeker25) [inline jobs/payment-jobs/services/__init__.py:15]: delete

File: jobs/payment-jobs/pyproject.toml [hunk 2]
```diff
@@ -28,6 +28,7 @@ launchdarkly-server-sdk = "^8.2.1"
 cx-oracle = "^8.3.0"
```
Comment (seeker25) [inline jobs/payment-jobs/pyproject.toml:28]: remove this please

File: jobs/payment-jobs/invoke_jobs.py [hunk 3]
```diff
@@ -58,8 +58,9 @@ def create_app(
     app.logger.info("<<<< Starting Payment Jobs >>>>")
     queue.init_app(app)
     db.init_app(app)
-    if init_oracle:
-        oracle_db.init_app(app)
+    if init_data_warehouse:
```
Comment (seeker25) [inline jobs/payment-jobs/invoke_jobs.py:61]: when wouldn't you init for datawarehouse?

Comment (Jxio) [reply jobs/payment-jobs/invoke_jobs.py:61]: some jobs no need init for datawarehouse.
```
    jobs_with_data_warehouse_connections = ["BCOL_REFUND_CONFIRMATION"]
    application = create_app(job_name=job_name, init_data_warehouse=job_name in jobs_with_data_warehouse_connections)
```

Comment (seeker25) [reply jobs/payment-jobs/invoke_jobs.py:61]: I think it's fine to just run it for any job

**Comment by Jxio** [comment]
New Key to add: DW_UNIX_SOCKET, DW_NAME, DW_USER, DW_PASSWORD, DW_HOST, DW_PORT

**Review by seeker25 — APPROVED** [review]

**Pull request merged**